### PR TITLE
added query DB Security Group Lower than 256

### LINKED ISSUE #1527: Add  DB Security Group Lower than 256 query for Ansible
### Platform
Ansible
### Provider
AWS

### Description
DB Security CIDR must be lower than 256 (/24)

### PR DESCRIPTION
**Pull request by cataraujo190**
Close #1527

### CODE REVIEW

File: assets/queries/ansible/aws/db_security_group_higherThan256/metadata.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "id": "db_security_group_lower_than_256",
+  "queryName": "DB Security Group Lower than 256",
+  "severity": "HIGH",
+  "category": "Network Security",
+  "descriptionText": "DB Security CIDR must be lower than 256 (/24)",
+  "descriptionUrl": "https://docs.ansible.com/ansible/latest/collections/community/aws/rds_instance_module.html#parameter-db_security_groups"
```
Comment (joaomartinscx) [inline assets/queries/ansible/aws/db_security_group_higherThan256/metadata.json:7]: Shouldn't be this resource instead? https://docs.ansible.com/ansible/latest/collections/amazon/aws/ec2_group_module.html#ansible-collections-amazon-aws-ec2-group-module

File: assets/queries/ansible/aws/db_security_group_higherThan256/query.rego [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  count(elements) > 0
+  values := concat(",", {e|e:=elements[p].cidr_ip;true})
+	result := {
+                "documentId": 		input.document[i].id,
+                "searchKey": 	    sprintf("name={{%s}}.rules_egress.cidr_ip", [ec2_instanceName]),
+                "issueType":		"IncorrectValue",
+                "keyExpectedValue": sprintf("'name={{%s}}.rules_egress.cidr_ip' is one of [10.0.0.0/8, 192.168.0.0/16, 172.16.0.0/12]", [ec2_instanceName]),
+                "keyActualValue": sprintf("'name={{%s}}.rules_egress.cidr_ip' is [%s]", [ec2_instanceName, values])
+              }
+}
```
Comment (joaomartinscx) [inline assets/queries/ansible/aws/db_security_group_higherThan256/query.rego:38]: According to the Terraform query you're supposed to check ingress traffic and not egress, why is it different now?

**Review by joaomartinscx — CHANGES_REQUESTED** [review]

File: assets/queries/ansible/aws/db_security_group_higherThan256/test/negative.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+- name: example ec2 group2
+  ec2_group1:
```
Comment (rogeriopeixotocx) [inline assets/queries/ansible/aws/db_security_group_higherThan256/test/negative.yaml:2]: invalid module name. `ec2_group1` should be `ec2_group`

File: assets/queries/ansible/aws/db_security_group_higherThan256/test/positive.yaml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+        # the containing group name may be specified here
+        group_name: example
+    rules_egress:
+      - proto: tcp
+        from_port: 80
+        to_port: 80
+        cidr_ip: 0.0.0.0/0
+        group_name: example-other
+        # description to use if example-other needs to be created
+        group_desc: other example EC2 group
```
Comment (rogeriopeixotocx) [inline assets/queries/ansible/aws/db_security_group_higherThan256/test/positive.yaml:57]: [As a good POSIX practice](https://unix.stackexchange.com/a/18789) add end-of-file newline

File: assets/queries/ansible/aws/db_security_group_higherThan256/test/negative.yaml [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+        to_port: 80
+        cidr_ip: 10.1.1.1/32
+  rules_egress:
+      - proto: tcp
+        from_port: 80
+        to_port: 80
+        cidr_ip: 10.1.1.1/32
+        group_name: example-other
+        # description to use if example-other needs to be created
+        group_desc: other example EC2 group
```
Comment (rogeriopeixotocx) [inline assets/queries/ansible/aws/db_security_group_higherThan256/test/negative.yaml:21]: [As a good POSIX practice](https://unix.stackexchange.com/a/18789) add end-of-file newline

File: assets/queries/ansible/aws/db_security_group_higherThan256/metadata.json [hunk 6]
```diff
@@ -0,0 +1,8 @@
...
+}
```
Comment (rogeriopeixotocx) [inline assets/queries/ansible/aws/db_security_group_higherThan256/metadata.json:8]: [As a good POSIX practice](https://unix.stackexchange.com/a/18789) add end-of-file newline

**Review by rogeriopeixotocx — CHANGES_REQUESTED** [review]

**Review by rogeriopeixotocx — APPROVED** [review]
LGTM

**Pull request merged**